maintenance/ebpf/ebpf-and-eks.md

```diff
@@ -184,26 +176,141 @@ which is suitable:
+  In this example, you would use `d881b853ae9313e00302a84f1e346a77.gr7.us-west-2.eks.amazonaws.com` for `KUBERNETES_SERVICE_HOST`
   and `443` (the default for HTTPS) for `KUBERNETES_SERVICE_PORT` when creating the config map.
+
+  Create the following config map in the `kube-system` namespace using the host and port determined above:
+
+  ```
+  kubectl apply -f - <<EOF
+  kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: kubernetes-services-endpoint
+    namespace: kube-system
+  data:
+    KUBERNETES_SERVICE_HOST: "<API server host>"
+    KUBERNETES_SERVICE_PORT: "443"
+  EOF
+  ```
+
+* Wait 60s for kubelet to pick up the `ConfigMap` (see Kubernetes [issue #30189](https://github.com/kubernetes/kubernetes/issues/30189){:target="_blank"}); then, restart the {{site.prodname}} pods to pick up the change:
+
+  ```
+  kubectl delete pod -n kube-system -l k8s-app=calico-node
+  ```
+
+  And, if using Typha and/or calico-kube-controllers (if you're not sure if you're running these, run the commands
+  anyway, they will fail with "No resources found" if the pods aren't present):
+  ```
+  kubectl delete pod -n kube-system -l k8s-app=calico-typha
+  kubectl delete pod -n kube-system -l k8s-app=calico-kube-controllers
+  ```
+
+* Confirm that pods restart and then reach the `Running` state with the following command:
+
+  ```
+  watch "kubectl get pods -n kube-system | grep calico"
+  ```
+
+  You can verify that the change was picked up by checking the logs of one of the  {{ site.nodecontainer }} pods.
+
+  ```
+  kubectl get po -n kube-system -l k8s-app=calico-node
+  ```
+
+  Should show one or more pods:
+
+  ```
+  NAME                                       READY   STATUS    RESTARTS   AGE
+  {{site.noderunning}}-d6znw                          1/1     Running   0          48m
+  ...
+  ```
+
+  Then, to search the logs, choose a pod and run:
+
+  ```
+  kubectl logs -n kube-system <pod name> | grep KUBERNETES_SERVICE_HOST
+  ```
+
+  You should see the following log, with the correct `KUBERNETES_SERVICE_...` values.
+
+  ```
+  2020-08-26 12:26:29.025 [INFO][7] daemon.go 182: Kubernetes server override env vars. KUBERNETES_SERVICE_HOST="172.16.101.157" KUBERNETES_SERVICE_PORT="6443"
+  ```
+
+#### Disable kube-proxy
+
+In eBPF mode {{site.prodname}} replaces `kube-proxy` so it wastes resources to run both.  To disable `kube-proxy` reversibly, we recommend adding a node selector to `kube-proxy`'s `DaemonSet` that matches no nodes, for example:
+
+```
+kubectl patch ds -n kube-system kube-proxy -p '{"spec":{"template":{"spec":{"nodeSelector":{"non-calico": "true"}}}}}'
+```
+
+Then, should you want to start `kube-proxy` again, you can simply remove the node selector.
+
+#### Enable eBPF mode
+
+To enable eBPF mode, change Felix configuration parameter  `BPFEnabled` to `true`.  This can be done with `calicoctl`, as follows:
```

Comment:
To enable eBPF mode, change the Felix configuration parameter `BPFEnabled` to `true`.